fabfile/db/__init__.py

```diff
@@ -159,8 +198,8 @@ def backup_db_sqlite(self, suffix, *args, **kwargs):
         """
         with lcd(self.base_path), settings(hide('warnings'), warn_only=True):
             local(
-                'cp %s/mysite.db %s/backups/backup.sqlite.%s'
-                % (dj_settings.BASE_DIR, self.base_path, suffix)
+                'cp %s/%s %s/backup.sqlite.%s'
```

Comment:
> Maybe we don't need user here at all, as we already have it in pgpass file?

self.db_cfg['NAME'] is a database name. Why are you talking about user at this point?